src/Nethermind/Nethermind.JsonRpc/JsonRpcProcessor.cs

```diff
@@ -225,7 +225,7 @@ JsonRpcResult GetParsingError(ref readonly ReadOnlySequence<byte> buffer, string
                 // Handles a single JSON RPC request.
                 if (model is not null)
                 {
-                    if (_logger.IsDebug) _logger.Debug($"JSON RPC request {model}");
+                    if (_logger.IsTrace) _logger.Trace($"JSON RPC request {model}");
```

Comment:
All the log in `JsonRpcProcessor` here should be debug.

Reply:
What if when in Debug mode RPC will be flushed but to separate files?